src/RealtimeEffectPanel.cpp

```diff
@@ -511,31 +511,31 @@ namespace
             }
          });
 
-         //Central button with effect name
-         const auto changeButton = safenew ThemedAButtonWrapper<AButton>(this, wxID_ANY);
-         changeButton->SetImageIndices(0,
+         //Central button with effect name, show settings
+         const auto optionsButton = safenew ThemedAButtonWrapper<AButton>(this, wxID_ANY);
+         optionsButton->SetImageIndices(0,
```

Comment:
I was not aware that this interchange of behavior of the two buttons was specified, but it is certainly correctly implemented.